code/renderers/vue3/src/render.ts

```diff
@@ -73,3 +98,25 @@ function getSlots(props: Args, context: StoryContext<VueRenderer, Args>) {
 
   return Object.fromEntries(slots);
 }
+
+/**
+ *  update the reactive args
+ * @param reactiveArgs
+ * @param nextArgs
+ * @returns
+ */
+function updateArgs(reactiveArgs: Args, nextArgs: Args) {
+  if (!nextArgs) return;
+  Object.keys(reactiveArgs).forEach((key) => {
+    delete reactiveArgs[key];
+  });
+  Object.assign(reactiveArgs, nextArgs);
+}
+
+function teardown(
+  storybookApp: ReturnType<typeof createApp>,
+  canvasElement: VueRenderer['canvasElement']
+) {
+  storybookApp?.unmount();
+  if (map.has(canvasElement)) map.delete(canvasElement);
+}
```

Comment:
[The Chromatic tests](https://www.chromatic.com/test?appId=630513346a8e284ae244d415&id=63cda8c7ca2f3db4f73babaf) shows that force re-rendering doesn't work anymore. The story focuses the button, emits a "force re-render" event and expects the button to loose focus, but it doesn't.

I wonder if that is a bug in this code here. I _think_ that it could be fixed if you unmounted the `storybookApp` from the `map` instead of from the parameter. So something like this (untested):
```suggestion
function teardown(
  canvasElement: VueRenderer['canvasElement']
) {
  if(!map.has(canvasElement)){
    return;
  }
  const { vueApp } = map.get(canvasElement);
  vueApp.unmount();
  map.delete(canvasElement);
}
```

Reply:
> [The Chromatic tests](https://www.chromatic.com/test?appId=630513346a8e284ae244d415&id=63cda8c7ca2f3db4f73babaf) shows that force re-rendering doesn't work anymore. The story focuses the button, emits a "force re-render" event and expects the button to loose focus, but it doesn't.
> 
> I wonder if that is a bug in this code here. I _think_ that it could be fixed if you unmounted the `storybookApp` from the `map` instead of from the parameter. So something like this (untested):

I don't think that is the issue, cause that is the exact reference I'm passing can't fail should be removed.

Reply:
you're right, it looks like there's actually something wrong with that story, that is unrelated to the work you're doing here. thanks for bringing it to my attention, I'm investigating if it is a regression.